lib/resty/acme/autossl.lua

```diff
@@ -423,14 +437,35 @@ function AUTOSSL.ssl_certificate()
   end
 end
 
+function AUTOSSL.create_account_key()
+  local t = ngx.now()
+  local pkey = util.create_pkey(4096, 'RSA')
+  ngx.update_time()
+  log(ngx_INFO, ngx.now() - t,  "s spent in creating new account key")
+  return pkey
+end
+
+function AUTOSSL.load_account_key_storage()
+  local storage = AUTOSSL.storage
+  local pkey, err = storage:get(account_private_key_prefix)
+  if err then
+    error("Failed to read account key from storage: " .. err)
+    return nil
+  end
+
+  if not pkey then
```

Comment:
this part of logic seems weird and `generated_key` seems unset

Reply:
Whops good catch.